website/src/pages/index.js

```diff
@@ -54,17 +54,17 @@ function Home() {
       title="Hermes"
       description="JavaScript engine optimized for React Native"
     >
-      <header className={classnames('hero hero--primary', styles.heroBanner)}>
+      <header className={classnames("hero hero--primary", styles.heroBanner)}>
         <div className="container">
           <h1 className="hero__title">{siteConfig.title}</h1>
           <p className="hero__subtitle">{siteConfig.tagline}</p>
           <div className={styles.buttons}>
             <Link
               className={classnames(
-                'button button--secondary button--lg',
-                styles.getStarted,
+                "button button--secondary button--lg",
```

Comment:
Can you undo these quotes changes? To keep the blame small

Reply:
done!